src/psyclone/dynamo0p3.py

```diff
@@ -4825,6 +4832,9 @@ def _initialise_xyoz_qr(self, parent):
             const_mod_list = self._invoke.invokes.psy. \
                 infrastructure_modules[const_mod]
             if kind not in const_mod_list:
+                # Record that we will need to import the kind for a
+                # declaration (associated with quadrature) from the
```

Comment:
s/quadrature/basis functions/ ?

Reply:
I think this is associated with quadrature. We add the precision (kind) associated with `gh_quadrature_xyoz`.